spec/javascripts/AuthenticationProviders/components/AccountAuthenticationProviders.spec.tsx

```diff
@@ -43,17 +44,17 @@ afterAll(() => {
 describe('when SSO toggle is hidden', () => {
   const props = { showToggle: false }
 
-  it('should only render a table', () => {
+  it('should render empty state', () => {
     const wrapper = shallowWrapper(props)
     expect(wrapper.exists(EnforceSSOSwitch)).toEqual(false)
-    expect(wrapper.exists(AuthenticationProvidersTable)).toEqual(true)
+    expect(wrapper.exists(AuthenticationProvidersEmptyState)).toEqual(true)
   })
 })
 
 describe('when SSO toggle is visible', () => {
-  const props = { showToggle: true }
+  const props = { showToggle: true, table: { ...defaultProps.table, count: 1 } }
 
-  it('should render both a switch and a table', () => {
+  it('should render both a switch and empty state', () => {
```

Comment:
It was correct before. It should show a table.
```suggestion
  it('should render both a switch a table, () => {
```